cmake/FDBComponents.cmake

```diff
@@ -252,7 +245,7 @@ function(print_components)
   message(STATUS "Build Documentation (make html):      ${WITH_DOCUMENTATION}")
   message(STATUS "Build Python sdist (make package):    ${WITH_PYTHON_BINDING}")
   message(STATUS "Configure CTest (depends on Python):  ${WITH_PYTHON}")
-  message(STATUS "Build with RocksDB:                   ${WITH_ROCKSDB_EXPERIMENTAL}")
+  message(STATUS "Build with RocksDB:                   ${SSD_ROCKSDB_EXPERIMENTAL}")
```

Comment:
nit: Is it possible to name it as `WITH_ROCKSDB`, we have passed the experimental phase :)